src/api/websocket/socketServer.js

```diff
@@ -8,13 +8,46 @@ let listCountries = countryPageMock;
 // Create an list of sockets by type
 const sockets = {
   home: [],
-  country: [],
+  country: {
+    list: [],
+    connections: 0,
+  },
   controller: [],
 };
 
 // Create an offSet for controller
 let offSetController = 0;
 
+// Controll the sending of events via socket about offset
+let isInTransition = false;
+
+/**
+ * This function is responsible for send information about offset
+ * on country page using socket with namespaces.
+ *
+ * @param {int} value value of offset
+ * @author Gabriel Nogueira
+ */
+async function sendingOffSet(value) {
+  // If another event already was sent then we return function to await
+  console.log(sockets.country.connections);
```

Comment:
`console.log()` aqui é intencional mesmo?

Reply:
Bem observado @VitorGois, acabei deixando passar esse console log.

@ryanraul nas próximas modificações que você for realizar a respeito das tasks de front end, poderia por gentileza excluir essa linha do `console.log()`?